sbt-app/src/sbt-test/classloader-cache/jni/build.sbt

```diff
@@ -10,7 +10,7 @@ val wrappedTest = taskKey[Unit]("Test with modified java.library.path")
 def wrap(task: InputKey[Unit]): Def.Initialize[Task[Unit]] =
   Def.sequential(appendToLibraryPath, task.toTask(""), dropLibraryPath)
 
-ThisBuild / turbo := true
+// ThisBuild / turbo := true
```

Comment:
No. Here's the result I get with turbo mode:

```scala
[info] -------------------------------- Running Tests --------------------------------
[info] X sbt.JniLibraryTest.load.native 79ms
[info]   java.lang.IllegalAccessError: tried to access class sbt.JniLibrary from class sbt.JniLibraryTest$
[info]     sbt.JniLibraryTest$.$anonfun$tests$3(JniLibraryTest.scala:11)
[info] [error] Test suite sbt.JniLibraryTest failed with java.lang.IllegalAccessError: tried to access class sbt.JniLibrary from class sbt.JniLibraryTest$.
[info] [error] This may be due to the ClassLoaderLayeringStrategy (AllLibraryJars) used by your task.
[info] [error] To improve performance and reduce memory, sbt attempts to cache the class loaders used to load the project dependencies.
[info] [error] The project class files are loaded in a separate class loader that is created for each test run.
[info] [error] The test class loader accesses the project dependency classes using the cached project dependency classloader.
[info] [error] With this approach, class loading may fail under the following conditions:
[info] [error]
[info] [error]  * Dependencies use reflection to access classes in your project's classpath.
[info] [error]    Java serialization/deserialization may cause this.
[info] [error]  * An open package is accessed across layers. If the project's classes access or extend
[info] [error]    jvm package private classes defined in a project dependency, it may cause an IllegalAccessError
[info] [error]    because the jvm enforces package private at the classloader level.
[info] [error]
[info] [error] These issues, along with others that were not enumerated above, may be resolved by changing the class loader layering strategy.
[info] [error] The Flat and ScalaLibrary strategies bundle the full project classpath in the same class loader.
[info] [error] To use one of these strategies, set the  ClassLoaderLayeringStrategy key
[info] [error] in your configuration, for example:
[info] [error]
[info] [error] set root / Test / classLoaderLayeringStrategy := ClassLoaderLayeringStrategy.ScalaLibrary
[info] [error] set root / Test / classLoaderLayeringStrategy := ClassLoaderLayeringStrategy.Flat
[info] [error]
[info] [error] See ClassLoaderLayeringStrategy.scala for the full list of options.
[info] [info] Tests: 1, Passed: 0, Failed: 1
[info] [error] Failed tests:
[info] [error] 	sbt.JniLibraryTest
[info] [error] (Test / testOnly) sbt.TestsFailedException: Tests unsuccessful
```